### ISSUE TITLE
How to read token cryptogram?

### ISSUE THREAD

**Issue by atonamy** [devnied/EMV-NFC-Paycard-Enrollment#24]
For EMV Payment Tokenisation (such as Android Pay and Apple Pay) instead of real card number providing token by NFC. This library successful  read token number and token expiry date. But how to read cryptogram. Where is stored? Any idea?

**Comment by spacemonkey92** [comment]
Any update on this?

**Comment by rozzalia** [comment]
Yes cryptogram posible a hard nutt but Posible

**Comment by sverraest** [comment]
@rozzalia Can you point us in the right direction?

**Comment by rozzalia** [comment]
Wil do little later not on my laptop now... Iff y don't get answers before
i have it back upp im in india my Acer did blow cpu Wil have it upp running
today hopefully ore tomorrow but this is big thing to crack i think so
happy to help

**Comment by sverraest** [comment]
@rozzalia were you able to share an example on getting the cryptogram from tokenized cards such as Apple Pay or Google Pay?

**Comment by rozzalia** [comment]
Yes so sorry i wil do when coming back tomorrow i totally forgot it last
time... Dont use alot off computer when travling
